iidm/iidm-extensions/src/main/java/com/powsybl/iidm/network/extensions/ActivePowerControl.java

```diff
@@ -54,4 +56,25 @@ default String getName() {
      */
     void setParticipationFactor(double participationFactor);
 
+    /**
+     * @return if present, provides the overridden value of pmin to be used for active power control operations.
+     */
+    Optional<Double> getMinPOverride();
+
+    /**
+     * Sets the overridden minimal active power.
+     * @param pMinOverride  The overridden value of Pmin. A Nan value removes the override.
+     */
+    void setMinPOverride(double pMinOverride);
+
+    /**
+     * @return if present, provides the overridden value of pmax to be used for active power control operations.
```

Comment:
pmax -> Pmax

Reply:
well, I corrected with maxP / minP, using the Generator API convention as well as the java bean convention. Hope it works.

Reply:
Just in doc !

Reply:
maxP / minP used in the doc as agreed.